lua/class_definitions.lua

```diff
@@ -2284,6 +2287,12 @@ global_functions = {
         cpp_name = "get_omt_dir",
         args = { "overmap", "tripoint" },
         rval = "overmap_direction"
+    },
+    omap_choose_point = {
+        cpp_name = "ui::omap::choose_point",
+        args = { },
+        optional_args = { "tripoint" },
```

Comment:
What's with optional arg? It doesn't seem to get used:

```C++
static int global_omap_choose_point(lua_State *L) {
    LuaValue<tripoint>::push(L, ui::omap::choose_point());
    return 1; // 1 return values
}
```

Reply:
Oh, it makes no sense. Fixed.